debian/rules

```diff
@@ -23,5 +27,36 @@ override_dh_auto_install:
 	rake --trace install
 	rake --trace install:nginx
 	rake --trace install:passenger
+
+	# install gems
 	mkdir -p $(DESTDIR)/opt/ood/gems
 	mv $(GEM_HOME)/* $(DESTDIR)/opt/ood/gems
+
+	# make some directories
+	mkdir -p "$(APACHE_DIR)/public"
+	mkdir -p "$(APACHE_DIR)/discover"
+	mkdir -p "$(APACHE_DIR)/register"
+	mkdir -p "$(APACHE_DIR)/apps/sys"
+	mkdir -p "$(APACHE_DIR)/apps/usr"
+	mkdir -p "$(APACHE_DIR)/apps/dev"
+	mkdir -p "$(CONFIG_DIR)/clusters.d"
+	mkdir -p "$(CONFIG_DIR)/ondemand.d"
+
+	mv $(DESTDIR)/opt/ood/apps/* $(APACHE_DIR)/apps/sys
+	rm -rf $(DESTDIR)/opt/ood/apps
+
+	# ood config files
+	rake package:version > $(DESTDIR)/opt/ood/VERSION
+	install -m 644 nginx_stage/share/nginx_stage_example.yml $(CONFIG_DIR)/nginx_stage.yml
+	install -m 640 ood-portal-generator/share/ood_portal_example.yml $(CONFIG_DIR)/ood_portal.yml
+	install -D -m 644 hooks/hook.env.example $(CONFIG_DIR)/hook.env
+	touch $(CONFIG_DIR)/ood_portal.sha256sum 
+
+	# extra infrastructure files
+	install -D -m 440 lib/tasks/files/sudo.debian $(DESTDIR)/etc/sudoers.d/ood
```

Comment:
I think a better location for these files would be `packaging/deb`.  Or do we want to defer that so we can also move RPM to `packaging/rpm`?  I think keeping all the packaging stuff out of lib in terms of packaged files seems logical.

Reply:
Yea, let's defer it until it's merge into master, then we can refactor both.

Reply:
I maybe misunderstood what you meant but either way if we want it now or later, opened #1428 to move RPM files.